glean-core/android/src/main/java/mozilla/telemetry/glean/net/Upload.kt

```diff
@@ -27,7 +27,7 @@ enum class UploadTaskTag {
 @Structure.FieldOrder("tag", "documentId", "path", "body", "headers")
 internal class UploadBody(
     // NOTE: We need to provide defaults here, so that JNA can create this object.
-    @JvmField val tag: Byte = UploadTaskTag.Done.ordinal.toByte(),
+    @JvmField val tag: Byte = UploadTaskTag.Upload.ordinal.toByte(),
```

Comment:
Why was it necessary to change the default here?

Reply:
It wasn't. I just noticed it when I wrote the `WaitBody` and IMO it felt wrong to have a tag of `Done` on an `UploadBody`.
We need default values, but won't actually use a default-constructed `UploadBody` further, so this change is not really noticable.